src/stirling/source_connectors/socket_tracer/uprobe_symaddrs.cc

```diff
@@ -142,19 +142,33 @@ StatusOr<std::string> InferHTTP2SymAddrVendorPrefix(ElfReader* elf_reader) {
   return vendor_prefix;
 }
 
+std::optional<int64_t> ResolveSymbolWithEachGoPrefix(ElfReader* elf_reader,
```

Comment:
Do we know the go version?
If so, should we:
1. only check `go.` if the go version is < 1.20?
2. reverse the order in the array, i.e. if v >= 1.20, check `go:` first?

Reply:
We could get the go version from the `runtime.buildVersion` symbol, (ala `ReadBuildVersion`). Unfortunately, the way this file is setup we'd have to repeatedly call that function in a number of places and then pipe it through, since there's no governing object or any other structure we can store the go version in, without a significant refactor. So my preference would be to keep it simple for now, and someone can potentially do the go version optimization later.